docs/configuration.md

```diff
@@ -2955,6 +2955,12 @@ Spark uses [log4j](http://logging.apache.org/log4j/) for logging. You can config
 `log4j.properties` file in the `conf` directory. One way to start is to copy the existing
 `log4j.properties.template` located there.
 
+By default, Spark adds 1 record to the MDC: `taskName`, which shows something
```

Comment:
do we have the full name of `MDC`? maybe some users don't know very well about it and we should give them something that they can get the general meaning.

Reply:
Done